app/src/main/java/com/li/musicmp3/data/repository/MusicRepository.kt

```diff
@@ -0,0 +1,21 @@
+package com.li.musicmp3.data.repository
+
+import com.li.musicmp3.data.source.MusicDataSource
+import com.li.musicmp3.data.source.local.MusicLocalDataSource
+import com.li.musicmp3.data.source.local.OnLoadMusicCallBack
+
+class MusicRepository private constructor(
+    private val localDataSource: MusicLocalDataSource
```

Comment:
Giao tiếp với nhau qua interface 
```suggestion
    private val localDataSource: MusicDataSource
```